app/recordtransfer/management/commands/send_test_email.py

```diff
@@ -0,0 +1,156 @@
+            "email_id", type=str, help="The ID of the email to send (email function name)."
+        )
+
+    def handle(self, *args, **options) -> None:
+        """Handle the command to send a test email.
+
+        Args:
+            *args: Variable length argument list.
+            **options: Arbitrary keyword arguments containing 'to_email' and 'email_id'.
+        """
+        to_email = options["to_email"]
+        email_id = options["email_id"]
+
+        if email_id not in EMAIL_FUNCTIONS:
+            raise CommandError(
+                f"Unknown email_id '{email_id}'. Valid options: {', '.join(EMAIL_FUNCTIONS.keys())}"
+            )
+
+        user = self.get_test_user(to_email)
+        form_data = {"dummy": "data"}
+        submission = self.create_test_submission(user)
+        in_progress = self.create_test_in_progress_submission(user)
+
+        try:
+            self.send_email(email_id, user, form_data, submission, in_progress)
+            logger.info("✅ Sent '%s' email to %s", email_id, to_email)
+        except Exception as e:
+            logger.exception("Error sending email '%s' to %s: %s", email_id, to_email, e)
+            raise CommandError(f"Error sending email: {e}") from e
+
+    def get_test_user(self, email: str) -> User:
+        """Retrieve or create a test user with the given email address.
+
+        Args:
+            email (str): The email address of the test user.
+
+        Returns:
+            User: The retrieved or newly created User instance.
+        """
+        user, _ = User.objects.get_or_create(
+            username="testuser",
+            defaults={
+                "email": email,
+                "first_name": "Test",
+                "last_name": "User",
+                "is_active": True,
+                "gets_notification_emails": True,
+                "gets_submission_email_updates": True,
+            },
+        )
+        return user
+
+    def create_test_submission(self, user: User) -> Submission:
+        """Create and return a test Submission instance for the given user.
+
+        Args:
+            user (User): The user for whom the submission is created.
+
+        Returns:
+            Submission: The created Submission instance.
+        """
+        return Submission.objects.create(
+            user=user,
+            raw_form=b"",
+            bag_name="test-bag",
+        )
+
+    def create_test_in_progress_submission(self, user: User) -> InProgressSubmission:
+        """Create and return a test InProgressSubmission instance for the given user."""
+        upload_session = UploadSession.objects.create(
+            started_at=timezone.now(),
+            status=UploadSession.SessionStatus.CREATED,
+            last_upload_interaction_time=timezone.now() - timedelta(minutes=10),
+        )
+        return InProgressSubmission.objects.create(
+            user=user,
+            title="In-Progress Test",
+            upload_session=upload_session,
+        )
```

Comment:
Same comment as above, we don't want to actually create an upload session or an in-progress submission in the database.

```suggestion
        upload_session = UploadSession(
            started_at=timezone.now(),
            status=UploadSession.SessionStatus.CREATED,
            last_upload_interaction_time=timezone.now() - timedelta(minutes=10),
        )
        return InProgressSubmission(
            user=user,
            title="In-Progress Test",
            upload_session=upload_session,
        )
```